src/BuildingBlocks/Caching/Masa.BuildingBlocks.Caching/IMultilevelCacheClient.cs

```diff
@@ -5,10 +5,196 @@ namespace Masa.BuildingBlocks.Caching;
 
 public interface IMultilevelCacheClient : ICacheClient
 {
+    /// <summary>
+    /// Get cache
+    /// When the memory cache does not exist, get the result of the distributed cache and store the result in the memory cache (the validity period of the memory cache is the expiration time passed in)
+    /// </summary>
+    /// <param name="key">cache key</param>
+    /// <param name="absoluteExpirationRelativeToNow">Absolute Expiration，Permanently valid when null</param>
+    /// <param name="action"></param>
+    /// <typeparam name="T"></typeparam>
+    /// <returns></returns>
+    T? Get<T>(string key, TimeSpan? absoluteExpirationRelativeToNow, Action<CacheOptions>? action = null);
+
+    /// <summary>
+    /// Get cache
+    /// When the memory cache does not exist, get the result of the distributed cache and store the result in the memory cache (the validity period of the memory cache is the expiration time passed in)
+    /// </summary>
+    /// <param name="key"></param>
+    /// <param name="absoluteExpiration"></param>
+    /// <param name="action"></param>
+    /// <typeparam name="T"></typeparam>
+    /// <returns></returns>
+    T? Get<T>(string key, DateTimeOffset? absoluteExpiration, Action<CacheOptions>? action = null);
+
+    /// <summary>
+    /// Get cache
+    /// When the memory cache does not exist, get the result of the distributed cache and store the result in the memory cache (the validity period of the memory cache is the expiration time passed in)
+    /// </summary>
+    /// <param name="key"></param>
+    /// <param name="memoryCacheEntryOptions"></param>
+    /// <param name="action"></param>
+    /// <typeparam name="T"></typeparam>
+    /// <returns></returns>
+    T? Get<T>(string key, CacheEntryOptions? memoryCacheEntryOptions, Action<CacheOptions>? action = null);
+
+    /// <summary>
+    /// Get cache
+    /// When the memory cache does not exist, get the result of the distributed cache and store the result in the memory cache (the validity period of the memory cache is the expiration time passed in)
+    /// </summary>
+    /// <param name="key"></param>
+    /// <param name="valueChanged"></param>
+    /// <param name="action"></param>
+    /// <typeparam name="T"></typeparam>
+    /// <returns></returns>
     T? Get<T>(string key, Action<T?> valueChanged, Action<CacheOptions>? action = null);
```

Comment:
Actually, I don't need to focus on the value changed event most of the time.

Reply:
So you think ValueChange is last?

Reply:
It inherits ICacheClient, ICacheClient provides access to cache information only through Key